Add Thaumcraft and Biomes O' Plenty planks and slightly adjust planks for vanilla woods

JohnButlerThird opened the pull request:
Currently:
- Biomes O' Plenty
- Thaumcraft

The wood don't get registered if the mod is not there.

serenibyss commented:
Could you describe what changed in the existing textures you modified? The new blocks look great, but I'm not sure why many existing ones are also changing in this PR

JohnButlerThird commented:
> Could you describe what changed in the existing textures you modified? The new blocks look great, but I'm not sure why many existing ones are also changing in this PR

I made them like the updated 1.12 textures, dark oak in chisel is overly saturated and doesnt look like dark oak. So I recolored to make it look more consistent with the new wood blocks.
They are just minor changes. If you want I can tweak it a bit more.

JohnButlerThird commented:
I just removed accidental dependencie of salis arcana

Alexdoru commented:
Your Pr says you are adding new blocks for modded woods, turns out you are also editing the textures of the current blocks.

These two changes should be two separate pull requests.

JohnButlerThird commented:
> Your Pr says you are adding new blocks for modded woods, turns out you are also editing the textures of the current blocks.
> 
> These two changes should be two separate pull requests.

Again I don't think it's that big of a deal, the base textures for woods in chisel are broken. I fixed them alongside this one.
Do you want me to do a PR without those changes? I can restore the base textures but I don't see the point.

Alexdoru commented:
> > Your Pr says you are adding new blocks for modded woods, turns out you are also editing the textures of the current blocks.
> > These two changes should be two separate pull requests.
> 
> Again I don't think it's that big of a deal, the base textures for woods in chisel are broken. I fixed them alongside this one. Do you want me to do a PR without those changes? I can restore the base textures but I don't see the point.

it is best practice to make separate additions into separate pull requests, I am not against any of your changes, it's just that's it makes it a mess to modify so many files in the same pull requests where you do two clearly different things

JohnButlerThird commented:
So do I need to revert texture change for this to be accepted?

Dream-Master commented:
@Alexdoru

Alexdoru commented:
> So do I need to revert texture change for this to be accepted?

it would be great if your pr only did what it says and that is "adding mod wood compat" if you want to change the existing textures make it in another separate pr please

JohnButlerThird commented:
> @JohnButlerThird @Alexdoru any news here?

my PR is pretty well made, I don't know why nobody is merging it.

wlhlm commented:
Hi, thanks for working on this. Nobody has merged it, because this pull request comes with two separate changes:

1. textures for BoP and Thaumcraft woods
2. retexture for vanilla chisel planks

The title of this pull request only referred to the first change. This is bad, because our changelogs are based on PR titles, so half of the work wouldn't even be listed.

Alexdoru has been pretty clear that they preferred this to be split into two separate pull requests, that's why not a lot has happened. I'm gonna be more lenient here. I have updated the pull request title to more accurately reflect the changes therein.

OK, here is how the textures look ingame. First, the BoP and Thaumcraft variants:
<img width="2560" height="1387" alt="2025-09-20_00 04 40" src="https://github.com/user-attachments/assets/fc20d8c1-ad00-4886-8324-852f363c8a7e" />

Overall nice. I do miss the color variation from BoP blanks for Hellbark, Magic, and Willow. They are replaced by just a "flat" color on the chisel planks. Also, I'm not sure about the Sacred Oak planks, the vines are maybe a bit too prominent. Looks quite repetitive when layed out on a larger surface. But it's not a big issue overall.
<img width="2560" height="1387" alt="2025-09-19_23 58 49" src="https://github.com/user-attachments/assets/b64b1b9b-fce2-44a2-8292-4861e76edeb4" />

And second the vanilla wood plank retexture:
<img width="2560" height="1387" alt="new-wood-comparison" src="https://github.com/user-attachments/assets/2a492e2d-7035-401c-b3fd-20fd88e1f582" />

I agree that the chisel planks for vanilla woods are improved quite a bit. However, is it intentionally that plank \# 7 for Acacia, Dark Oak, and Spruce look just like plank \# 1?

The java code changes all work. Tested with full pack GTNH, the textures are correctly loaded.

BTW, you need to update your branch before we can merge it.

JohnButlerThird commented:
> Hi, thanks for working on this. Nobody has merged it, because this pull request comes with two separate changes:
> 
>     1. textures for BoP and Thaumcraft woods
> 
>     2. retexture for vanilla chisel planks
> 
> 
> The title of this pull request only referred to the first change. This is bad, because our changelogs are based on PR titles, so half of the work wouldn't even be listed.
> 
> Alexdoru has been pretty clear that they preferred this to be split into two separate pull requests, that's why not a lot has happened. I'm gonna be more lenient here. I have updated the pull request title to more accurately reflect the changes therein.
> 
> OK, here is how the textures look ingame. First, the BoP and Thaumcraft variants: <img alt="2025-09-20_00 04 40" width="2000" height="1387" src="https://private-user-images.githubusercontent.com/1928113/491821284-fc20d8c1-ad00-4886-8324-852f363c8a7e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgzMjYwNDAsIm5iZiI6MTc1ODMyNTc0MCwicGF0aCI6Ii8xOTI4MTEzLzQ5MTgyMTI4NC1mYzIwZDhjMS1hZDAwLTQ4ODYtODMyNC04NTJmMzYzYzhhN2UucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MTlUMjM0OTAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YmEwYjRiZWJhODNiMmNjN2JmM2I5NDljYzQ2ZWEwODQ4NDJiMjM3OTRhYzY1MGNhZDM5NGJjZDU3OTE5ZjlmYSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.TWHLpktf4zuxMO0ioNU_JDKa-KBb73AcOOCVadMey1U">
> 
> Overall nice. I do miss the color variation from BoP blanks for Hellbark, Magic, and Willow. They are replaced by just a "flat" color on the chisel planks. Also, I'm not sure about the Sacred Oak planks, ~the vines are maybe a bit too prominent. Looks quite repetitive when layed out on a larger surface. But it's not a big issue overall.~ I guess it just follows the original BoP texture...
> 
> And second the vanilla wood plank retexture: <img alt="new-wood-comparison" width="2000" height="1387" src="https://private-user-images.githubusercontent.com/1928113/491821402-2a492e2d-7035-401c-b3fd-20fd88e1f582.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTgzMjYwNDAsIm5iZiI6MTc1ODMyNTc0MCwicGF0aCI6Ii8xOTI4MTEzLzQ5MTgyMTQwMi0yYTQ5MmUyZC03MDM1LTQwMWMtYjNmZC0yMGZkODhlMWY1ODIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkxOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MTlUMjM0OTAwWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YmE0M2QxN2IwMjhhZGE0NmFkY2I1ZDQyYjkwMmU4ZmE0Njc4YzUyZDkxZWJiMmY0Y2IxZjY3ODM2ZGQyODc4OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.WOaArcIZIewOcBTHHitxisC9MgIE2X9Xwb2d2PAL6F0">
> 
> I agree that the chisel planks for vanilla woods are improved quite a bit. However, is it intentionally that plank # 7 for Acacia, Dark Oak, and Spruce look just like plank # 1?
> 
> The java code changes all work. Tested with full pack GTNH, the textures are correctly loaded.
> 
> BTW, you need to update your branch before we can merge it.

YEah I got that, I just didnt know people cared that much. I changed the vanilla textures because they fucking sucked especially dark oak that looks absolutely disgusting. Doing a seperate PR just for textures changes was just not something i thought was necessary. I can tweak a bit of the textures I changed, but they are basically the one from 1.12.

wlhlm commented:
OK, I'm unfamiliar with the 1.12 textures. We can leave it if it's more or less a backport. I was just asking in case there was a mixup.

Ushupiuck commented:
> Hi, thanks for working on this. Nobody has merged it, because this pull request comes with two separate changes:
> 
>     1. textures for BoP and Thaumcraft woods
> 
>     2. retexture for vanilla chisel planks
> 
> 
> The title of this pull request only referred to the first change. This is bad, because our changelogs are based on PR titles, so half of the work wouldn't even be listed.
> 
> Alexdoru has been pretty clear that they preferred this to be split into two separate pull requests, that's why not a lot has happened. I'm gonna be more lenient here. I have updated the pull request title to more accurately reflect the changes therein.
> 
> OK, here is how the textures look ingame. First, the BoP and Thaumcraft variants: <img alt="2025-09-20_00 04 40" width="2000" height="1387" src="https://private-user-images.githubusercontent.com/1928113/491821284-fc20d8c1-ad00-4886-8324-852f363c8a7e.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTkxODQzMzksIm5iZiI6MTc1OTE4NDAzOSwicGF0aCI6Ii8xOTI4MTEzLzQ5MTgyMTI4NC1mYzIwZDhjMS1hZDAwLTQ4ODYtODMyNC04NTJmMzYzYzhhN2UucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MjlUMjIxMzU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MTdkZDMwNmYwNTU0NTUxOTk3M2U2ZTU3ZDVjOGEzY2MxZDczNzljY2UyNjYyZTY0NjFjZGY2MDliNjA1ZjBjNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.AGdxTNstmKAGjuKQS-7qaMGa9vJoST-qod9426YghNA">
> 
> Overall nice. I do miss the color variation from BoP blanks for Hellbark, Magic, and Willow. They are replaced by just a "flat" color on the chisel planks. Also, I'm not sure about the Sacred Oak planks, ~the vines are maybe a bit too prominent. Looks quite repetitive when layed out on a larger surface. But it's not a big issue overall.~ I guess it just follows the original BoP texture...
> 
> And second the vanilla wood plank retexture: <img alt="new-wood-comparison" width="2000" height="1387" src="https://private-user-images.githubusercontent.com/1928113/491821402-2a492e2d-7035-401c-b3fd-20fd88e1f582.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NTkxODQzMzksIm5iZiI6MTc1OTE4NDAzOSwicGF0aCI6Ii8xOTI4MTEzLzQ5MTgyMTQwMi0yYTQ5MmUyZC03MDM1LTQwMWMtYjNmZC0yMGZkODhlMWY1ODIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI1MDkyOSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNTA5MjlUMjIxMzU5WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9YWU0OWU4YTFhNTA3YjY0MzE0ZjkyZjJiZTNmMzIxYzMxOTZkN2FiMmRiMDkxNTUzNTViYWYyMGY2MWNkOTYzNCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.Ws0jzxuvS1iqNNXVmrxwV8cLB6NXd6vFzGH9PjElMM8">
> 
> I agree that the chisel planks for vanilla woods are improved quite a bit. However, is it intentionally that plank # 7 for Acacia, Dark Oak, and Spruce look just like plank # 1?
> 
> The java code changes all work. Tested with full pack GTNH, the textures are correctly loaded.
> 
> BTW, you need to update your branch before we can merge it.

Comparing the old vs new textures... Some of the "old" textures were somewhat more grainy; Detailed. The new textures are just -- to be raw? Flat looking; The same base texture, recolored. This is especially noticeable with the first two textures.

To be blunt, and feel free to take offense, but... Unless you can point out improvements, this just looks like a glorified texture smoothening without flavor.

Ushupiuck commented:
Tonal differences within the color itself, making the detail stand out more -- compare new 5 to old 1/2 and you'll understand.

JohnButlerThird commented:
> I feel like the Acacia and Dark Oak ones were different enough that maybe there should be a config option for those who liked the old ones better.

The old ones were saturated and looked like shit, this just fixed the inconstency and made acacia and dark oak look like their actual wood.

JohnButlerThird commented:
> > The old ones were saturated and looked like shit, this just fixed the inconstency and made acacia and dark oak look like their actual wood.
> 
> I mean, that's your opinion, but I for one liked the old ones better. Specifically because they stand out more. Adding options wouldn't hurt.

The old one didn't even look like acacia/dark oak. This is just for consistency.
